packages/govuk-frontend/src/govuk/components/fieldset/_index.scss

```diff
@@ -35,24 +35,27 @@
   }
 
   // Modifiers that make legends look more like their equivalent headings
+  .govuk-fieldset__legend--xl,
+  .govuk-fieldset__legend--l,
+  .govuk-fieldset__legend--m {
+    @include govuk-typography-weight-bold;
```

Comment:
Can we save some more code and stick `$weight: bold` in `.govuk-fieldset__legend` instead?

Reply:
Same thing as https://github.com/alphagov/govuk-frontend/pull/4267#discussion_r1341530600